src/edu/csus/ecs/pc2/ui/PluginLoadPane.java

```diff
@@ -178,47 +178,42 @@ public void setContestAndController(IInternalContest inContest, IInternalControl
     private JPanePlugin[] getPluginList() {
         Vector<JPanePlugin> plugins = new Vector<JPanePlugin>();
         
-        plugins.add(new RunsTablePane());
-        plugins.add(new AccountsTablePane());
-        plugins.add(new StandingsTablePane());
-        plugins.add(new ClarificationsTablePane());
-        plugins.add(new EventFeedServerPane());
-        plugins.add(new AccountsTablePane());
         plugins.add(new AccountsPane());
```

Comment:
Expected to see a change with 5 fewer lines/Panes, aka remove duplicated panes.
As a result of the way this was changed I am required to manually determine determine
if extra Panes were removed or added.

Reply:
In the future to avoid this confusion, push two commits.
1 - that fixes the problem without any scope creep/CI
2 - the scope creep (sorting of array)

Reply:
I have no problem with the rearrangement of the code (so that the list of additions is in alphabetical order); that is precisely the kind of "continuous improvement" that will help avoid a repetition of the problem (the existence of duplicate entries) in the future.  I also agree that while it's not (strictly speaking) part of the "Issue" which was filed, it's very reasonable to include it _as part of the PR_, under the banner of "CI" (because it's the kind of thing that would NEVER get done if it was shuffled off to it's own separate "Issue".)

Having said that, I also agree with Doug that it would have been very helpful if the "original issue" part (removal of the duplicates and the "CI" part (rearranging the list into alphabetical order to make it easier in the future to avoid duplicates)  had been submitted in separate commits (while still part of the same PR).  As it was, it turned this PR from a "5 minute review/approval" into one that took -- literally -- over three months, the reason being that I looked at it twice before, once in June and once again in July, and both times I hit the situation that I realized it was going to take way more than 5 minutes to review so I ended up putting it back in the "queue of things to do".  (In fact, it took me over half an hour to verify that (a) all the correct duplicates had been removed; (b) no incorrect entries had been removed, and (c) all the correct entries were still present in the "reshuffled" list.  I'm pretty sure I could have done that same verification in 5 minutes if there had been separate commits.)

Having said _that_, I will freely admit that _I'm not certain that I **myself** would have thought to make two separate commits when doing this_.   Hopefully we'll all get better at realizing how to make things easier... :)